client/src/partials/templates/bhFindDocument.tmpl.html

```diff
@@ -1,137 +1,128 @@
-<div class="patient-document">
+<!-- Patient details informations  -->
+<div class="document-section" ng-if="$ctrl.session.enablePatientDetails">
+  <b><i class="fa fa-universal-access"></i> {{ $ctrl.session.patient.reference }} / </b>
+  {{ $ctrl.session.patient.first_name + ' ' + $ctrl.session.patient.last_name + ' ' + $ctrl.session.patient.middle_name }}
```

Comment:
As a side note: we originally used `$scope.session` before we used `Controller As` syntax.  Writing `$ctrl.session = {}` should be seen as deprecated going forward.